filecoin-proofs/src/api/mod.rs

```diff
@@ -314,7 +314,15 @@ where
         .labels
         .verify_stores(verify_store, &cache)?;
```

Comment:
The labels path here also needs to be updated, or I can modify the verify to only use the provided cache dir, rather than cross checking.  I'll look at the latter option for now.